tests/files/c_source/motohawk_include_signal_names.c

```diff
@@ -0,0 +1,175 @@
+/**
```

Comment:
this, and the corresponding in this PR, seems to be spurious: It should be generated as part of running the tests?